styles/crunchyroll/catppuccin.user.css

```diff
@@ -893,7 +901,7 @@
         .clickable-background(@surface1);
 
         &.state-active {
-          color: @accent-color;
+          color: @accent-color !important;
```

Comment:
I do not need `!important` here in both Firefox & Chromium.

Reply:
mb